packages/data-service-generator/src/tests/__snapshots__/create-data-service-custom-path.spec.ts.snap

```diff
@@ -475,8 +478,8 @@ import { StringFilter } from \\"../../util/StringFilter\\";
 import { StringNullableFilter } from \\"../../util/StringNullableFilter\\";
 import { BooleanNullableFilter } from \\"../../util/BooleanNullableFilter\\";
 import { DateTimeNullableFilter } from \\"../../util/DateTimeNullableFilter\\";
-import { FloatNullableFilter } from \\"../../util/FloatNullableFilter\\";
-import { IntNullableFilter } from \\"../../util/IntNullableFilter\\";
+import { DecimalNullableFilter } from \\"../../util/DecimalNullableFilter\\";
```

Comment:
Should this really be "FloatNullableFilter"? do we have such file?

the file FloatNullableFilter is based on the type "number" and can be used both for decimal and float. Also I cant find the new file in the static folder